apigw-rest-api-http-integration-terraform/README.md

```diff
@@ -0,0 +1,51 @@
+# Amazon API Gateway REST API with HTTP integration in Terraform
+
+This Terraform template deploys an Amazon API Gateway REST API endpoint with a simple public HTTP endpoint integration.
+
+Learn more about this pattern at Serverless Land Patterns: [https://serverlessland.com/patterns/apigw-rest-api-http-integration-terraform](https://serverlessland.com/patterns/apigw-rest-api-http-integration-terraform)
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [Terraform Installed](https://developer.hashicorp.com/terraform/downloads)
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+2. Change directory to the pattern directory:
+    ```
+    cd apigw-rest-api-http-integration-terraform
+    ```
+3. From the command line, run:
+    ```
+    terraform init
+    ```
+4. From the command line, run:
+    ```
+    terraform plan
```

Comment:
apply and plan are asking for a HTTP URL value, that is not described in the installation steps. 
Can you modify the steps or the code to match. 
```
$ terraform apply --auto-approve
var.GetHttpUrl
  The HTTP URL to integrate with AWS API Gateway

  Enter a value: 
```

Reply:
Thanks for the feedback @mavi888.
I have updated the README.md file  with the instructions on how to test.

Reply:
@mavi888 Could you please check and update. Thank you!

Reply:
The deployment is still asking for an URL when doing terraform plan and terraform apply. 
Can you please add this in the instructions.

```
$ terraform plan
var.GetHttpUrl
  The HTTP URL to integrate with AWS API Gateway

  Enter a value: 
```